cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -191,3 +210,39 @@ func normalizeDotNetSecret(key string) string {
 	// ":".
 	return strings.ReplaceAll(key, "__", ":")
 }
+
+// findProjectFile locates the project file to pass to the `dotnet` tool for a given dotnet service. In the case where
+// [Path] for the the service results in a directory, if current directory contains a single file matching the
+// glob `*.*proj`, then project file will be returned. If there are multiple or no project file, error is returned
```

Comment:
nit: Add a newline between the summary and description if you want it show up nicer in VSCode:

// <summary>
//
// <description>


Which will show up like:
```
<summary text>
<description text>
```

instead of:

```
<summary text> <description text>
```